x-pack/metricbeat/module/containerd/memory/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Same here, sample events must be as representative as possible since are supposed to provide good documentation.